.github/workflows/template-connect-popup-test-params.yml

```diff
@@ -92,16 +100,24 @@ jobs:
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
 
-      - name: Build webextension
+      - name: Install dependencies
         run: |
           yarn install --immutable
           yarn build:libs
+
+      - name: Build webextension
+        run: |
           yarn workspace @trezor/connect-webextension build
           yarn workspace @trezor/connect-iframe build:core-module
           yarn workspace @trezor/connect-explorer build:webextension
 
-      - name: Make Docker Script Executable
-        run: chmod +x ./docker/docker-connect-popup-ci.sh
+      - name: Build example webextension
+        env:
+          URL: https://${{ inputs.DEV_SERVER_HOSTNAME }}/connect/${{ steps.extract_branch.outputs.branch }}/
+        run: |
+          yarn workspace @trezor/connect-web build:webextension
+          yarn workspace @trezor/connect-web build:inline
+          node packages/connect-examples/update-webextensions.js --trezor-connect-src "${URL}" --npm-src "${URL}trezor-connect.js"
```

Comment:
so url here is just regular branch of a PR right?  (or maybe develop branch)

also didn't we use it somewhere in combination production URLs?

Reply:
Yes, URL here is that, the branch of the PR where this test is running. On schedule it is develop.